### ISSUE TITLE
Outdated scipy gaussian import

### ISSUE THREAD

**Issue by wulfdewolf** [JorritMontijn/zetapy#15]
The following files use:
https://github.com/JorritMontijn/zetapy/blob/18333298bab40390d0f35868bcf0044204cabafb/zetapy/plot_dependencies.py#L13
https://github.com/JorritMontijn/zetapy/blob/18333298bab40390d0f35868bcf0044204cabafb/zetapy/legacy/dependencies.py#L7

In the [latest stable scipy release v1.13.0](https://docs.scipy.org/doc/scipy/index.html) the gaussian function has moved to [`scipy.signal.windows`](https://docs.scipy.org/doc/scipy-1.13.0/reference/signal.windows.html#module-scipy.signal.windows).

So either the imports should become:
```
from scipy.signal import convolve
from scipy.signal.windows import gaussian
```
or an explicit scipy version should be set in the pip dependencies, [v0.14.0](https://docs.scipy.org/doc/scipy-0.14.0/reference/generated/scipy.signal.gaussian.html) is the latest one I found that still has it the old way.

**Comment by JorritMontijn** [comment]
Thanks! I'll fix it when I next update the repo

**Comment by guidomeijer** [comment]
This is fixed in the new release 3.0.7
